packages/core/test/Cardano/util/estimateStakePoolAPY.test.ts

```diff
@@ -0,0 +1,39 @@
+import { Cardano } from '../../../src';
+
+describe('estimateStakePoolAPY', () => {
+  const rewards = {
+    epochLength: 432_000_000,
+    stakingROI: 0.000_68
```

Comment:
Where did you source this value?

Reply:
Just calculated it, this sums up to ~5% APY with 5 day epochs